scripts/python-skywater-pdk/skywater_pdk/netlistsvg-generate.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Maybe `skywater_pdk/cells/generate/netlist.py`?

Reply:
This is being handled in https://github.com/google/skywater-pdk/pull/247